android-components/components/browser/state/src/test/java/mozilla/components/browser/state/reducer/InternetResourceReducerUtilsTest.kt

```diff
@@ -0,0 +1,20 @@
+package mozilla.components.browser.state.reducer
+
+import mozilla.components.browser.state.state.BrowserState
+import mozilla.components.browser.state.state.ContentState
+import mozilla.components.browser.state.state.TabSessionState
+import org.junit.Assert
+import org.junit.Test
+
+class InternetResourceReducerUtilsTest {
+    @Test
+    fun `updateTheContentState will return a new BrowserState with updated ContentState`() {
+        val initialContentState = ContentState("emptyStateUrl")
+        val browserState = BrowserState(tabs = listOf(TabSessionState("tabId", initialContentState)))
+
+        val result = updateTheContentState(browserState, "tabId") { it.copy(url = "updatedUrl") }
+
+        Assert.assertFalse(browserState == result)
+        Assert.assertEquals("updatedUrl", result.tabs[0].content.url)
```

Comment:
The `Assert.assertFalse` /  `Assert.assertEquals` should be imported.

Reply:
Good spot - automatic import 🙃